ui/helpers/job.js

```diff
@@ -173,6 +173,8 @@ export const getResultState = function getResultState(job) {
 
 export const addAggregateFields = function addAggregateFields(job) {
   const {
+    // eslint-disable-next-line no-unused-vars
+    hasSideBySide,
```

Comment:
If this variable is not used then why was it added? We should aim to avoid disabling linting rules, these were set for a reason, to keep the code clean.